packages/api/api-generator/src/commands/generate/generateCrud/__tests__/filter-primary-key.spec.ts

```diff
@@ -37,6 +37,9 @@ describe("filter primary key", () => {
             const foundFile = formattedOut.find((file) => file.name === "test-entity.resolver.ts");
             if (!foundFile) throw new Error("File not found");
         });
+        afterEach(async () => {
+            await orm.close();
+        });
```

Comment:
Why did you decide to add an afterEach hook here and simply add await everywhere else?

Reply:
when `orm` is created in `beforeEach`, `close` should be called in `afterEach`. When `orm` is created in a single test ("inline"), `close` should also be done inline.